src/main/java/frc/robot/constants/RobotComponents.java

```diff
@@ -17,15 +18,16 @@
     }
 
     protected static class SwerveComponents {
-        public static final Pigeon PIGEON = new Pigeon(CAN.Swerve.PIGEON_ID);
+        public static final Pigeon PIGEON = new Pigeon(new TrigonTalonSRX(12));
```

Comment:
have it get a motor from a different subsystem and if it doesn't have it be
` new TrigonTalonSRX(CAN.Swerve.PIGEON_ID) `